package.json

```diff
@@ -1,65 +1,78 @@
+      "^.*[.]css$": "<rootDir>/utils/empty-modules.js"
     },
-    "jest": {
-        "rootDir": "./src/",
-        "moduleNameMapper": {
-            "^.*[.]css$": "<rootDir>/utils/empty-modules.js"
-        },
-        "setupFiles": ["<rootDir>/utils/enzyme-setup.js"]
-    },
-    "devDependencies": {
-        "babel-core": "^6.24.1",
-        "babel-jest": "^21.2.0",
-        "babel-loader": "^6.4.1",
-        "babel-plugin-transform-class-properties": "^6.24.1",
-        "babel-preset-env": "^1.6.0",
-        "babel-preset-es2015": "^6.24.1",
-        "babel-preset-react": "^6.24.1",
-        "chai": "^4.1.2",
-        "css-loader": "^0.28.0",
-        "d3-geo": "1.6.4",
-        "enzyme": "^3.0.0",
-        "enzyme-adapter-react-15": "^1.0.0",
-        "file-loader": "^0.11.1",
-        "jest": "^21.2.1",
-        "path": "0.12.7",
-        "react-test-renderer": "^15.5.3",
-        "react-addons-test-utils": "^15.4.2",
-        "serve-favicon": "2.4.2",
-        "style-loader": "^0.16.1",
-        "topojson-client": "3.0.0",
-        "url-loader": "^0.5.8",
-        "webpack": "^2.3.3",
-        "webpack-dev-server": "^2.7.1"
-    },
-    "dependencies": {
-        "axios": "^0.16.2",
-        "bootstrap": "^3.3.7",
-        "d3": "^4.11.0",
-        "fiber-css": "0.5.1",
-        "highcharts": "^5.0.14",
-        "html-webpack-plugin": "^2.30.1",
-        "material-ui": "^0.19.2",
-        "moment": "^2.0.0",
-        "normalize.css": "6.0.0",
-        "react": "^15.5.3",
-        "react-bootstrap": "^0.30.8",
-        "react-dom": "^15.5.3",
-        "react-moment": "^0.6.5",
-        "react-router-dom": "^4.2.2",
-        "react-tap-event-plugin": "^2.0.1",
-        "superagent": "^3.6.0"
-    }
+    "setupFiles": [
+      "<rootDir>/utils/enzyme-setup.js"
+    ]
+  },
+  "devDependencies": {
+    "babel-core": "^6.24.1",
+    "babel-eslint": "^8.0.1",
+    "babel-jest": "^21.2.0",
+    "babel-loader": "^6.4.1",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "chai": "^4.1.2",
+    "css-loader": "^0.28.0",
+    "d3-geo": "1.6.4",
+    "enzyme": "^3.0.0",
+    "enzyme-adapter-react-15": "^1.0.0",
+    "eslint": "^4.8.0",
+    "eslint-config-standard": "^10.2.1",
+    "eslint-config-standard-react": "^5.0.0",
+    "eslint-plugin-babel": "^4.1.2",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-node": "^5.2.0",
+    "eslint-plugin-promise": "^3.5.0",
+    "eslint-plugin-react": "^7.4.0",
+    "eslint-plugin-standard": "^3.0.1",
+    "file-loader": "^0.11.1",
+    "jest": "^21.2.1",
+    "path": "0.12.7",
+    "react-addons-test-utils": "^15.4.2",
+    "react-test-renderer": "^15.5.3",
+    "serve-favicon": "2.4.2",
+    "style-loader": "^0.16.1",
+    "topojson-client": "3.0.0",
+    "url-loader": "^0.5.8",
+    "webpack": "^2.3.3",
+    "webpack-dev-server": "^2.7.1"
+  },
+  "dependencies": {
```

Comment:
Did you forget to add semantic-ui-react dependency to this package.json? i couldn't find it here and the app couldnt be started because it couldnt find the lib.